packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -1,4 +1,20 @@
 import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import {
+  AUTOCOMPLETE_REQUEST,
+  AUTOCOMPLETE_RESPONSE,
+  AUTOCOMPLETE_ERROR,
+  SAYT_PRODUCTS_REQUEST,
+  SAYT_PRODUCTS_RESPONSE,
+  SAYT_PRODUCTS_ERROR,
+  AutocompleteRequestPayload,
+  AutocompleteResultGroup,
+  AutocompleteSearchTermItem,
+  AutocompleteResponsePayload,
+  AutocompleteErrorPayload,
+  SaytProductsRequestPayload,
+  SaytProductsResponsePayload,
+  SaytProductsErrorPayload,
+} from '@sfx/events';
```

Comment:
For readability and consistency where it's used. It's a lot easier to remember that you can use `AUTOCOMPLETE_REQUEST` everywhere and not have to remember that this file and that file have a wildcard import. In addition, listing all the imports explicitly makes it clear exactly which ones are being used in this file.

Reply:
True. On the other hand, proper namespacing can actually improve readability and decrease variable names. Eg. `SaytProductsRequestPayload` could be `ProductsRequestPayload` if it's used (for example) like: `sayt.ProductsRequestPayload`.

This can be resolved, but I wanted to poke at the approach here.